packages/report-server/src/app/addRoutesToApp.ts

```diff
@@ -15,5 +33,15 @@ export function addRoutesToApp(app) {
   /**
    * POST routes
    */
-  app.post('(/v[0-9]+)?/fetchReport/:reportCode', fetchReport);
+  app.post('(/v[0-9]+)?/fetchReport/:reportCode', wrappedFetchReportWithAsyncCatch);
+
+  app.use(handleError);
 }
+
+const handleError = (err, req: ReportsRequest, res: Response) => {
+  let error = err;
+  if (!error.respond) {
+    error = new InternalServerError(err);
+  }
```

Comment:
This is a way to avoid "let", although the existing code will also do!

```suggestion
  const error = err.respond ? err : new InternalServerError(err);
```